core/src/navigation/TopNav.html

```diff
@@ -19,8 +19,53 @@
       />
       {/if}
       {#if children && pathData.length > 0}
-      {#each children as node}
+      {#each children as node ,i}
       {#if !node.hideFromNav}
+      {#if node.isCat}
+      <div
+        class="fd-shellbar__action fd-shellbar__action--hide fd-shellbar__action--collapsible"
+      >
+        <div class="fd-product-menu">
+          <div class="fd-popover fd-popover--right">
+            <div class="fd-popover__control" on:click="event.stopPropagation()">
+              <button
```

Comment:
The Button has some weird padding on the right, guess this is also the reason that the popover is positioned to far right.
<img width="161" alt="screen markus e 2019-07-01 at 12 42 50" src="https://user-images.githubusercontent.com/1720843/60431787-e5274580-9c00-11e9-983b-39de249e9582.png">